.github/workflows/cluster-ui-release.yml

```diff
@@ -64,6 +64,13 @@ jobs:
         cp ../../../../_bazel/bin/pkg/ui/workspaces/db-console/src/js/protos.* ../db-console/src/js/
         yarn build
 
+    - name: Create version tag and push
+      if: steps.version-check.outputs.published == 'no'
+      run: |
+        TAGNAME="@cockroachlabs/cluster-ui@$(cat ./package.json | jq -r '.version')"
```

Comment:
super nit that I don't expect you to fix: `jq` accepts a filename!  This can be simplified to `jq -r '.version' ./package.json` for the next time you use `jq` 😃

Reply:
Nice! Thanks for the tip. I'll make a note to myself for next time.